temba/channels/types/novo/type.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import six
+
+from django.utils.translation import ugettext_lazy as _
+
+from temba.channels.models import ChannelType
+from temba.channels.types.novo.views import ClaimView
+from temba.contacts.models import TEL_SCHEME
+
+
+class NovoType(ChannelType):
+    """
+    A Novo channel (http://www.novotechnologyinc.com/)
+    """
+
+    CONFIG_MERCHANT_ID = "merchant_id"
+    CONFIG_MERCHANT_SECRET = "merchant_secret"
+
+    courier_url = r"^nv/(?P<uuid>[a-z0-9\-]+)/(?P<action>receive)$"
+
+    code = "NV"
+    category = ChannelType.Category.PHONE
+
+    name = "Novo"
+
+    claim_blurb = _(
+        """If you are based in Trinidad & Tobago, you can purchase a short code from <a href="http://www.novotechnologyinc.com/">Novo</a> and connect it in a few simple steps."""
+    )
```

Comment:
Pretty sure this replaces the claim template @norkans7

Reply:
That is show on the section for the channel type when we have all types listed,

the claim template is for the form

Reply:
Ohh right, the template is need for those that extends the AuthenticatedForm.